backend/src/v5/schemas/tickets/templates.js

```diff
@@ -187,6 +187,13 @@ const configSchema = Yup.object().shape({
 			.test('Custom status', 'values must be unique', (vals) => uniqueElements(vals.map(({ name }) => name)).length === vals.length),
 		default: Yup.mixed().when('values', (values) => (values ? Yup.string().oneOf(values.map(({ name }) => name)).required() : Yup.mixed())),
 	}).default(undefined),
+	tabular: Yup.object({
```

Comment:
I'd try to enforce the module-property validity test here so that it covers both create/edit a template, something similar to pinMappingTest